docs/protocol/extension_statistics.md

```diff
@@ -217,6 +226,22 @@ $duration_stat =
 
 - “ns” : The total duration for the statistic in nanoseconds.
 
+```
+$memory_usage = 
+{
+  "type" : $string,
+  "id" : $number,
+  "byte_size" : $number
+}
```

Comment:
Is `memory_usage` an array of `type`, `id`, `byte_size` values? Can a model report a mix of these values?

Reply:
Yes. For example, if model is deployed on every GPU on the system, then there will be entries matching the number of GPUs.